astropy/coordinates/tests/test_pickle.py

```diff
@@ -27,47 +32,24 @@ def test_pickle_longitude_wrap_angle():
     assert a.wrap_angle == b.wrap_angle
 
 
-_names = [
-    coord.Angle,
-    coord.Distance,
-    coord.DynamicMatrixTransform,
-    coord.ICRS,
-    coord.Latitude,
-    coord.Longitude,
-    coord.StaticMatrixTransform,
-]
-
-_xfail = [False, not HAS_SCIPY, True, True, False, True, False]
-
-_args = [
-    [0.0],
-    [],
-    [lambda *args: np.identity(3), coord.ICRS, coord.ICRS],
-    [0, 0],
-    [0],
-    [0],
-    [np.identity(3), coord.ICRS, coord.ICRS],
-]
-
-_kwargs = [
-    {"unit": "radian"},
-    {"z": 0.23},
```

Comment:
I don't think these two are properly ported in this refactor yet

Reply:
This is supposed to be a pickling test and these objects have no memory of how they were initialized.

Reply:
thanks for clarifying